CHANGELOG/CHANGELOG-2.0.md

```diff
@@ -0,0 +1,117 @@
+## Release notes for 2.0.0
+
+[Documentation](https://kubernetes-csi.github.io)
+
+## Urgent Upgrade Notes
+
+### (No, really, you MUST read this before you upgrade)
+
+- The minimum Kubernetes version is 1.34 for the storage/v1 VolumeAttributesClass API object. Emulated versions below 1.34 are supported but a restart is required once the emulated version is updated to 1.35.
```

Comment:
Should it contain user facing content rather?

"This version of resizer requires v1.34.0 of Kubernetes to support Volume modification via VolumeAttributeClass APIs". 

> Emulated versions below 1.34 are supported but a restart is required once the emulated version is updated to 1.35.
 
Emulated versions below v1.34 are supported but volume modification feature will be disabled in that case right? Should we explicitly say that?

Reply:
it's a little complex because it depends on if you have the beta API enabled or not.

Reply:
I added that to the urgent notes. Anything else before we merge the change log?